code/ui/blocks/src/controls/Boolean.tsx

```diff
@@ -112,16 +119,17 @@ export const BooleanControl: FC<BooleanProps> = ({ name, value, onChange, onBlur
   const parsedValue = typeof value === 'string' ? parse(value) : value;
 
   return (
-    <Label htmlFor={controlId} title={parsedValue ? 'Change to false' : 'Change to true'}>
+    <Label htmlFor={controlId} aria-label={name}>
```

Comment:
Does the title need to be removed? It added a tooltip which was useful for any user

Reply:
In fact, we can improve the title further and provide the name of the control which will be changed:
<img width="376" alt="image" src="https://github.com/storybookjs/storybook/assets/1671563/68fe43c1-458b-4b06-861e-f71802f88578">

And the voiceover (seems to be) even more helpful:
![image](https://github.com/storybookjs/storybook/assets/1671563/27466291-204e-4e15-8121-bf5a1819a26e)

While behaving similarly when interacting with the control:
![image](https://github.com/storybookjs/storybook/assets/1671563/b09de550-5483-4f49-82e0-5957f8d77a1f)

Reply:
Hi, @yannbf.

I have basically applied the [Switch Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/switch/), as described by the W3C Web Accessibility Initiative, which states that:

- It is critical the label on a switch does not change when its state changes
- The switch has an accessible label provided by one of the following:
  - Visible text content contained within the element with role switch.
  - A visible label referenced by the value of `aria-labelledby` set on the element with role switch.
  - `aria-label` set on the element with role switch.

The idea is letting the semantic elements and corresponding ARIA roles to let screen reader users know how to use the component, and having a stable label for the element that does not change based on the current value of the interactive element.

That is why I have removed the old labels "Change to true" and "Change to false", and just left a stable one, which is the name of the control which will be changed.

That is also the reason why I have removed the `title` attribute, as:
  - It was not necessary for sighted users, as there's enough visual context to understand its behavior
  - It was no longer necessary for screen reader users, as `aria-label` and `role="switch"` are already conveying the information on both its usage, and the control they relate with

There's also a small excerpt on MDN mentioning that "While title can be used to provide a programmatically associated label for an <input> element, this is not good practice. Use a <label> instead." You can read it [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title).

Knowing this, do you consider these changes to be okay?

Reply:
Hey @danielmarcano thanks a lot for all the clarification. I think it makes sense, let's proceed without the title!